elastic/kibana
# [Table list view] Integrate package in Dashboard

In this PR I've updated the dashboard listing view and replaced the `<TableListView />` component imported from the `kibana_react` plugin by the package one created in https://github.com/elastic/kibana/pull/138244

## Note on implementation

I've created a `<TableListViewKibanaProvider>` which plugin can use to provide the stateful dependencies to the component.

```js
<TableListViewKibanaProvider
  {...{
    core: coreStart,
    toMountPoint,
    savedObjectsTagging,
    FormattedRelative,
  }}
>
  <App />
</TableListViewKibanaProvider>
```

## How to test

There aren't any new functionality in this PR. We should have good functional tests and component integration test to prevent regression.

* Install the sample data
* Navigate to the Dashboard listing app
* Verify that everything works as expected
  * Tags are displayed
  * Tag dropdown filter works as expected
  *  Search works
  * (Bulk) delete works
  * Navigate to details view works
